lib/config/configuration.go

```diff
@@ -73,10 +73,10 @@ import (
 )
 
 const (
-	// LogFileDefaultMode is the preferred permissions mode for log file.
-	LogFileDefaultMode fs.FileMode = 0o644
-	// LogFileDefaultFlag is the preferred flags set to log file.
-	LogFileDefaultFlag = os.O_WRONLY | os.O_CREATE | os.O_APPEND
+	// logFileDefaultMode is the preferred permissions mode for log file.
+	logFileDefaultMode fs.FileMode = 0o644
+	// logFileDefaultFlag is the preferred flags set to log file.
+	logFileDefaultFlag = os.O_WRONLY | os.O_CREATE | os.O_APPEND
```

Comment:
Move the constants to a smaller package and import it on both lib/config and event-handler?

Reply:
_a little copying is better than a little dependency_

I don't think we need any new packages for this stuff. It's unlikely these constants ever change.

Reply:
> a little copying is better than a little dependency

That doesn't mean your own project, but OK.

Reply:
FWIW, `integrations/` is effectively a different project. It was only recently copied into this repo and there are still tons of things that we should rip out in favor of code we already have in Teleport. For example, I removed a some dead code in #47101 and captured #47102 with some more follow ups earlier this week.

For now I'd just like to fix the builds, but I do want to see us continue to clean up these integrations (most of them were originally written by contractors, so they don't follow many of our standards) going forward.